src/game/Tactical/Turn_Based_Input.cc

```diff
@@ -2413,12 +2395,6 @@ void GetKeyboardInput(UIEventKind* const puiNewEvent)
 			}
 			else if ( InputEvent.usKeyState & CTRL_DOWN )
       {
```

Comment:
ah, ok. I didn't notice the tabbing here. In this case I agree, let's solve that somewhere else.